gnupg-static/plan.sh

```diff
@@ -1,7 +1,16 @@
 source ../gnupg/plan.sh
 
 pkg_name=gnupg-static
+pkg_origin=core
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_version=1.4.20
+pkg_license=('GPL-3.0-or-later')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="GnuPG is a complete and free implementation of the OpenPGP standard as defined by RFC4880 (also known as PGP). \
+  This is a statically complied version of gnupg."
```

Comment:
complied should be compiled

In general, is it preferred in this repo to push trivial commits to PRs; or to feedback trivial comments and leave the PR author control of commits? I was going to add a commit to fix this typo but wanted to check.

Reply:
🤦‍♂️  ERR_NOT_ENOUGH_COFFEE.  I'll fix and squish. 

It's better to let the author push commits, IMO.  In many cases, the commits will be on a branch in a fork so you won't have write access.